toolkits/global/packages/global-forms/view/globalFormErrorSummary.hbs

```diff
@@ -7,13 +7,15 @@
 		</span>
 	</div>
 	<div class="c-forms__error-summary-text">
-		<div class="c-forms__error-summary-title" id="{{id}}">{{title}}</div>
-		<ul class="c-forms__error-summary-errors">
-			{{#each errors}}
-				<li>
-					<a href="#{{id}}">{{error}}</a>
-				</li>
-			{{/each}}
-		</ul>
+		<div class="c-forms__error-summary-title" id="{{id}}" tabindex="-1">{{title}}</div>
```

Comment:
We actually took the path to have the `tabindex="-1"` onto the aside element.
Any particular reason you enforce it on the title rather?

On our side that let the focus ring surrounds the aside rather than the title itself.

Reply:
@morgaan I put it there because you were have issues with announcement because it was not on a text element? You could use `:focus-within` to set the focus style on the container 🕶